lib/fs.ts

```diff
@@ -26,16 +26,23 @@ import * as pathModule from 'path';
 import {ReadStream, WriteStream} from './streams';
 
 const ROOT_PATH = pathModule.resolve(__dirname, '..');
+const DEFAULT_THROTTLE = 5 * 1000; // 5 seconds
 
-interface PendingUpdate {
+export interface PendingUpdate {
```

Comment:
```suggestion
interface PendingUpdate {
```
We ended up not exporting this, right?